doc/guides/getting_started/src/README.md

```diff
@@ -221,22 +194,22 @@ After installation you need to add `/tools/verible/$VERIBLE_VERSION/bin` to your
 
 Note that we currently use version {{#tool-version verible }}, but it is expected that this version is going to be updated frequently, since the tool is under active development.
 
-### Step 6b: Install OpenOCD (optional)
+### step 7b: Install OpenOCD (optional)
```

Comment:
Did you mean to make these lower case? Is it so that they are clearer as sub steps?

Reply:
Oops! Fixed now, thanks.